### PR TITLE
metrics(ticdc): etcd worker tick reactor duration also show p999 to reveal long duration ticks.

### LINKED ISSUE #6926: puller logs wrong message make user confused
```
[2022/08/30 14:32:54.840 +08:00] [WARN] [ddl_puller.go:357] ["ddl puller resolved ts has not advanced"] [namespace=default] [changefeed=blackhole-test_ddl_puller] [duration=2562047h47m16.854775807s] [resolvedTs=435640275400130814]
[2022/08/30 14:32:54.920 +08:00] [WARN] [ddl_puller.go:357] ["ddl puller resolved ts has not advanced"] [namespace=default] [changefeed=mysql-test_ddl_puller] [duration=2562047h47m16.854775807s] [resolvedTs=435634842599948608]
[2022/08/30 14:33:24.840 +08:00] [WARN] [ddl_puller.go:357] ["ddl puller resolved ts has not advanced"] [namespace=default] [changefeed=blackhole-test_ddl_puller] [duration=2562047h47m16.854775807s] [resolvedTs=435640275400130814]
[2022/08/30 14:33:24.920 +08:00] [WARN] [ddl_puller.go:357] ["ddl puller resolved ts has not advanced"] [namespace=default] [changefeed=mysql-test_ddl_puller] [duration=2562047h47m16.854775807s] [resolvedTs=435634842599948608]
```

### PR DESCRIPTION
**Pull request by 3AceShowHand**
### What problem does this PR solve?

Issue Number: close #6926, #6900 

### What is changed and how it works?

image

* remove `p95` metrics, add `p999` metrics

image

* add a new display for `changefeed tick duration`

#### Tests 

 - Manual test (add detailed scripts or steps below)

### Release note 

```release-note
`None`
```

### CODE REVIEW

File: metrics/grafana/ticdc.json [hunk 1]
```diff
@@ -12805,10 +12805,10 @@
           "targets": [
             {
               "exemplar": true,
-              "expr": "histogram_quantile(0.95, sum(rate(ticdc_etcd_worker_tick_reactor_duration_bucket{k8s_cluster=\"$k8s_cluster\", tidb_cluster=\"$tidb_cluster\", instance=~\"$ticdc_instance\"}[1m])) by (le, instance))",
+              "expr": "histogram_quantile(0.999, sum(rate(ticdc_etcd_worker_tick_reactor_duration_bucket{k8s_cluster=\"$k8s_cluster\", tidb_cluster=\"$tidb_cluster\", instance=~\"$ticdc_instance\"}[1m])) by (le, instance))",
```
Comment (overvenus) [inline metrics/grafana/ticdc.json:12808]: Please also attach `changefeed=~\"$changefeed\"`

Comment (3AceShowHand) [reply metrics/grafana/ticdc.json:12808]: it's a server level metrics, record `owner` and `processorManager`, not belong to any changefeed.

**Review by overvenus — APPROVED** [review]

**Review by asddongmen — APPROVED** [review]

**Pull request merged**